common/locales/de.json

```diff
@@ -92,9 +92,12 @@
     "camp": {
       "collaborators": {
         "guest": "Gast",
+        "guestAbilities": "Nur Leserecht",
         "invite": "Einladen",
-        "manager": "Verwalter",
-        "member": "Mitglied"
+        "manager": "Administration",
+        "managerAbilities": "Vollständige Lageradministration, sowie Lese- & Schreibrecht",
```

Comment:
do we need this in the common locales?

Reply:
I think it is useful, as it is not a sentence and more a state, that can be used in E-mails as well.

Reply:
But for emails we currently can only use the translations in the backend

Reply:
Okay. Would you prefer these keys in the frontend locales?